iaso/api/bulk_create_users.py

```diff
@@ -164,7 +164,9 @@ def create(self, request, *args, **kwargs):
                                         )
                             except ValueError:
                                 try:
-                                    org_unit = OrgUnit.objects.get(name=ou)
+                                    org_unit = OrgUnit.objects.get(
+                                        name=ou, pk__in=request.user.iaso_profile.org_units.all()
```

Comment:
I think this is not going to work like it should. We want to allow any org unit that are children (including themselves, as the awkard definition of children in computer science trees means it) of the ones in iaso_profile.org_units.all().  See calls to hiearchy() in some other places of the code base